### PR TITLE
changed folder structure

### PR DESCRIPTION
**Pull request by DIONYSUS-BASOUNAS**

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -7,10 +7,12 @@
             "args": [
                 "-fdiagnostics-color=always",
                 "-g",
-                "${workspaceFolder}\\*.cpp",
-                "${workspaceFolder}\\creature\\*.cpp",
-                "${workspaceFolder}\\creature\\main_character\\*.cpp",
-                "${workspaceFolder}\\start_menu\\*.cpp",
+
```
Comment (achilleasmitos) [inline .vscode/tasks.json:10]: Unnecessary empty line, please remove.

File: .vscode/tasks.json [hunk 2]
```diff
@@ -7,10 +7,12 @@
...
                 "-g",
-                "${workspaceFolder}\\*.cpp",
-                "${workspaceFolder}\\creature\\*.cpp",
-                "${workspaceFolder}\\creature\\main_character\\*.cpp",
-                "${workspaceFolder}\\start_menu\\*.cpp",
+              
+                "${workspaceFolder}\\scr\\*.cpp",
+                "${workspaceFolder}\\scr\\creature\\*.cpp",
+                "${workspaceFolder}\\scr\\utils\\*.cpp",
+                "${workspaceFolder}\\scr\\creature\\main_character\\*.cpp",
```
Comment (achilleasmitos) [inline .vscode/tasks.json:14]: Please keep the alphabetical order of these. See how it was done before.

**Review by achilleasmitos — CHANGES_REQUESTED** [review]

File: .vscode/tasks.json (hunk 1 above)
Comment (DIONYSUS-BASOUNAS) [reply .vscode/tasks.json:10]: done

File: .vscode/tasks.json (hunk 2 above)
Comment (DIONYSUS-BASOUNAS) [reply .vscode/tasks.json:14]: done

**Review by achilleasmitos — COMMENTED** [review]
Good job! Make sure to squash your commits into 1 before merging.

**Review by achilleasmitos — APPROVED** [review]

**Review by achilleasmitos — APPROVED** [review]

**Review by achilleasmitos — APPROVED** [review]
Noticed the lack of music during testing, which made me also notice the wrong name of the source folder. Pushed changes to fix this.

**Review by achilleasmitos — APPROVED** [review]

**Pull request merged**